AAI/AAIConnectProfile.md

```diff
@@ -194,12 +193,14 @@ the Broker.
     Flow (with Confidential Client)
     <http://openid.net/specs/openid-connect-basic-1_0.html>
 
-2.  Public Clients (typically javascript browser clients or mobile apps)
+2.  Public Clients (typically javascript browser clients)
     SHOULD implement OIDC Implicit Flow
     (<http://openid.net/specs/openid-connect-implicit-1_0.html>)
 
     1.  MUST use "id_token token" response_type for authentication.
 
+    2.  Public Clients like mobile apps that use Authorization Code Grant SHOULD implement PKCE (<https://tools.ietf.org/html/draft-ietf-oauth-spop-15>)
```

Comment:
This is a draft spec, so hard to link to it firmly until it is ratified. I would suggest changing what is in the brackets to be: "(such as <link> or an IETF specification that supersedes it)"